headers/types/dungeon_mode/dungeon_mode.h

```diff
@@ -1072,16 +1269,19 @@ struct fixed_room_properties_entry {
     struct music_id_16 music;
     undefined field_0x2;
     undefined field_0x3;
-    bool illuminated; // 0x4: Floor will be fully illuminated (darkness level DARKNESS_BRIGHT)
-    bool enable_lategame_traps; // 0x5: Allows Summon, Pitfall, and Pokémon traps to spawn
-    bool ai_moves_enabled;      // 0x6: NPCs can use moves on this floor
-    bool orbs_allowed;          // 0x7: Orbs can be used. Ignored for IDs past FIXED_SEALED_CHAMBER.
+    bool illuminated;           // 0x4: Floor will be fully illuminated (darkness level 0)
+    bool enable_lategame_traps; // 0x5: Allows Summon, Pitfall, and Pokémon traps to work
+    // 0x6: Monsters can use moves on this floor. Does not affect the regular attack.
+    // If false, NPCs won't use any moves at all (not even the regular attack)
+    bool moves_enabled;
```

Comment:
Does this affect the player as well? If not, I think the old name `ai_moves_enabled` still makes more sense.

Reply:
Yes, it does.

Reply:
Interesting, I didn't realize there were floors that disabled moves entirely. `moves_enabled` is better then.